elm-format-lib/src/Parse/Primitives.hs

```diff
@@ -202,36 +201,6 @@ instance Monad (Parser x) where
       parserA state cok' eok' cerr eerr
 
 
-
--- FROM BYTESTRING
-
-
---fromByteString :: Parser x a -> (Row -> Col -> x) -> B.ByteString -> Either x a
---fromByteString (Parser parser) toBadEnd (B.PS fptr offset length) =
---  B.accursedUnutterablePerformIO $
---    let
---      toOk' = toOk toBadEnd
---      !pos = plusPtr (unsafeForeignPtrToPtr fptr) offset
---      !end = plusPtr pos length
---      !result = parser (State fptr pos end 0 1 1) toOk' toOk' toErr toErr
---    in
---    do  touchForeignPtr fptr
---        return result
-
```

Comment:
This will eventually be needed; ideally https://github.com/avh4/elm-format/blob/main/elm-format-lib/src/ElmFormat/Parse.hs#L22 will change to take ByteString instead of Text, and https://github.com/avh4/elm-format/blob/aded67fb5db42afd17bfa372eab23f8a01cb635d/elm-format-lib/src/Parse/Parse.hs will change to take ByteString instead of String.

Sorry things are a bit of a mess atm.

Reply:
Specifically, https://github.com/avh4/elm-format/issues/301 is the blocking issue for the above mentioned ideal changes.